e2e_playwright/st_select_slider.py

```diff
@@ -18,11 +18,21 @@
 import streamlit as st
 from streamlit import runtime
 
+
+def format_option(option):
+    return f"Color: {option}"
+
+
 w1 = st.select_slider(
     "Label 1",
```

Comment:
nit: In many other widget tests we have been using the labels to give some indication of what the widgets is supposed to be tested for: 

https://github.com/streamlit/streamlit/blob/7f24fc068da25a919e7c96d307c444ce29890e98/e2e_playwright/st_text_input.py#L33